db/index.js

```diff
@@ -0,0 +1,12 @@
+const {Pool} = require("pg")
```

Comment:
The task had not yet required setting up this file yet - please ensure to only follow the instructions of the ticket in future PRs

Reply:
I had misunderstood the instructions thinking I was meant to setup an indexfile. will delete! thanks